draft-ietf-quic-transport.md

```diff
@@ -1894,10 +1894,10 @@ Note that Stateless Reset messages do not have any cryptographic protection.
 
 # Frame Types and Formats
 
-As described in {{frames}}, Regular packets contain one or more frames.
-We now describe the various QUIC frame types that can be present in a Regular
-packet. The use of these frames and various frame header bits are described in
-subsequent sections.
+As described in {{frames}}, packets contain one or more frames.  We now
```

Comment:
Not new text, but basically none of the recent text is written in the first-person plural.  Can we rephrase this, while you're touching the paragraph?